src/server/origin/on-off-farm/controller.js

```diff
@@ -39,11 +42,14 @@ export const onOffFarmPostController = {
       })
     }
 
-    req.yar.set('onOffFarm', onOffFarm)
+    req.yar.set('origin', {
+      ...req.yar.get('origin'),
+      onOffFarm
+    })
 
     switch (onOffFarm) {
       case 'off':
-        return res.redirect('/cph-number')
+        return res.redirect(nextPage ?? '/origin/cph-number')
```

Comment:
Same bug here as above - nextPage isn't going to be null / undefined in any circumstance, so we're getting a redirect to a blank location.